DOCS/man/mpv.rst

```diff
@@ -499,32 +499,53 @@ console controls. (Which makes it suitable for playing data piped to stdin.)
 The special argument ``--`` can be used to stop mpv from interpreting the
 following arguments as options.
 
-When using the client API, you should strictly avoid using ``mpv_command_string``
+For paths passed to mpv CLI options, the situation is further complicated by the
```

Comment:
this is not correct, the old docs intentionally talked about "suboptions", because only then you have a problem with `,` or `:`.
if you have a simple option that takes a single string (path), you can pass anything to it without problems.

Reply:
Ahh, I will rephrase to match the original explanation.